mypy/test/testfinegrained.py

```diff
@@ -90,7 +100,7 @@ def run_case_inner(self, testcase: DataDrivenTestCase, cache: bool) -> None:
             # cache, now we need to set it up
             if fine_grained_manager is None:
                 messages, manager, graph = self.build(main_src, testcase, sources_override,
-                                                      build_cache=False, enable_cache=cache)
+                                                      build_cache=False, enable_cache=True)
```

Comment:
Is the change to a `True` argument on purpose?

Reply:
Yeah. This path only executes when the cache is on.